cfg/defaultConstants.js

```diff
@@ -14,7 +16,7 @@ module.exports = {
 
   ticketRefreshIntervalInSeconds: 300,
 
-  authVersion: 'ADAL',
+  authVersion: 'TEST',
```

Comment:
Not sure if we want to do it this way; the default for running localhost should be ADAL. We only want TEST when actually running tests; the app should be completely non-functional when authVersion is TEST. Having to override configuration before launching the app for the first time gets in the way of 'clone and it just works' as a low barrier to entry on the project.

Reply:
synced offline.